gravitee-apim-console-webui/src/management/api/list/api-list.component.ts

```diff
@@ -31,30 +33,62 @@ type ApisTableDS = { id: string; name: string }[];
   template: require('./api-list.component.html'),
   styles: [require('./api-list.component.scss')],
 })
-export class ApiListComponent implements OnInit {
+export class ApiListComponent implements OnInit, OnDestroy {
   displayedColumns = ['name'];
-
   apisTableDSUnpaginatedLength = 0;
   apisTableDS: ApisTableDS = [];
   filters: GioTableWrapperFilters = {
     pagination: { index: 1, size: 10 },
     searchTerm: '',
   };
-  constructor(@Inject(UIRouterState) private readonly $state: StateService, private readonly apiService: ApiService) {}
+private unsubscribe$: Subject<boolean> = new Subject<boolean>();
+  private filters$ = new BehaviorSubject<GioTableWrapperFilters>(this.filters);
+
+  constructor(
+    @Inject(UIRouterStateParams) private $stateParams,
```

Comment:
Just a filling : 
This is not observable so use $ a the begin is confusing
Maybe `ajsStateParams` to be clear this comme from angularJs stuff 🤷‍♂️ 

(I think I'm the one who added the $ in the first 🙈 )

Reply:
Done